src/components/Widget/components/FullScreenPreview/styles.scss

```diff
@@ -0,0 +1,56 @@
+.fullscreen-container {
+  width: 100vw;
+  height: 100vh;
+  background: rgba(0, 0, 0, 0.75);
+  overflow: hidden;
+  position: fixed;
+  z-index: 9999;
+  left: 0;
+  top: 0;
+
+  &__image {
```

Comment:
I'm not sure about the nesting. This makes it a little more verbose to overwrite in case you need to change it, so would it be better without it. What do think?